htdocs/install/mysql/migration/15.0.0-16.0.0.sql

```diff
@@ -276,3 +276,13 @@ UPDATE llx_c_ticket_type SET use_default=1 WHERE code='OTHER' AND NOT EXISTS(SEL
 
 UPDATE llx_c_country SET eec=0 WHERE eec IS NULL;
 ALTER TABLE llx_c_country MODIFY COLUMN eec tinyint DEFAULT 0 NOT NULL;
+
+ALTER TABLE llx_c_availability ADD COLUMN type_duration char(1);
+ALTER TABLE llx_c_availability ADD COLUMN number real DEFAULT 0;
+
+delete from llx_c_availability WHERE rowid IN (1,2,3,4,5);
```

Comment:
Do not include instruction that may destroy customization of user into a migration. Instead,
just keep the insert (there is a protection of duplicate due to the unique key on code) and after, for each insert, make an update to update the new field when code = the code.
Do not rely on rowid that may be different on each installation.

Reply:
I made some modifications, are you ok with that ?